packages/ui/__tests__/EmptyState.test.tsx

```diff
@@ -0,0 +1,136 @@
+import { mountAndCheckA11Y } from '@hazelcast/test-helpers'
+import { Figma } from '@icons-pack/react-simple-icons'
+import React from 'react'
+import { LinkRel } from '../src/Link'
+import { EmptyState } from '../src/EmptyState'
+import { ChevronRight } from 'react-feather'
+
+// Common
+const title = 'Figma'
+const description =
+  'Figma is a vector graphics editor and prototyping tool, with additional offline features enabled by desktop applications for macOS and Windows.'
+// Icon
+const icon = Figma
+const iconLabel = 'Icon Figma'
+// Action
+const action = 'Cool!'
+const actionHref = ''
+const actionTarget = '_blank'
+const actionRel: LinkRel[] = ['noopener', 'noreferrer']
+
+describe('EmptyState', () => {
+  it('Renders', async () => {
+    const actionOnClick = jest.fn()
+
+    const wrapper = await mountAndCheckA11Y(
+      <EmptyState title={title} icon={icon} iconLabel={iconLabel} action={action} actionOnClick={actionOnClick} />,
+    )
+
+    expect(wrapper.findDataTest('empty-state-icon').props()).toMatchObject({
```

Comment:
nit: we can be more strict to ensure expected objects have the right structure.
```suggestion
    expect(wrapper.findDataTest('empty-state-icon').props()).toEqual<IconProps>({
```